data/content_purpose_supergroups.md

```diff
@@ -0,0 +1,20 @@
+# Content Purpose Supergroups
```

Comment:
Can these docs be added to the YAML file? That way they'll show up in the docs.

Reply:
Sure. I can do that.